src/ContentBlocks/BlockImage/BlockImage.js

```diff
@@ -23,8 +24,9 @@ export default function BlockImage({ content, alt, image, borderColor }) {
         src={path}
         alt={alt}
       />
-
-      {caption}
+      <Box margin={{ horizontal: 'none', vertical: 'small' }} pad="none">
+        {caption}
+      </Box>
```

Comment:
This is already fixed. We need to be sure there's a caption there before adding padding to keep things consistent. https://github.com/grommet/grommet-cms-content-blocks/blob/master/src/ContentBlocks/BlockImage/styles.js#L16-L21